Source/DSPSpy/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+cmake_minimum_required(VERSION 3.5.0)
+
+project(dspspy)
+
+set(CMAKE_EXECUTABLE_SUFFIX ".elf")
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
+
+set(MACHDEP "-DHW_RVL -DRVL -DGEKKO -mrvl -mcpu=750 -meabi -mhard-float")
```

Comment:
Is there any reason this isn't just placed in a call for `target_compile_definitions` for the -D flags, and `target_compile_options` for the others instead of modifying the build flags directly?

Reply:
To be honest: I don't really know. This part was directly copied from the CMakeLists.txt in the hwtests repo, and that's how the flags were set. I agree this doesn't make a lot of sense; I'll change it to use `target_compile_{definitions,options}`.

Reply:
Oh right, with `target_compile_definitions`, you also omit the `-D` part for flags specified in it (forgot to clarify that in the first comment).